casbin/core_enforcer.py

```diff
@@ -75,7 +92,7 @@ def init_with_model_and_adapter(self, m, adapter=None):
 
     def _initialize(self):
         self.rm_map = dict()
-        self.eft = get_effector(self.model["e"]["e"].value)
+        self.eft = get_effector(self.model["e"][self.etype].value)
```

Comment:
@techoner I think because the etype is being used here, that we still have to make etype as a class variable, so we might have just leave them all as class variables?